client/attest_network_test.go

```diff
@@ -23,7 +24,7 @@ func TestNetworkFetchIssuingCertificate(t *testing.T) {
 
 	key := &Key{cert: akCert}
 
-	certChain, err := key.getCertificateChain()
+	certChain, err := key.getCertificateChain(&http.Client{})
```

Comment:
In this file, we should have a 

```go
var externalClient = http.DefaultClient
```

We can then define the client to be a different value inside google3